front/app/services/participationContexts.ts

```diff
@@ -66,10 +67,11 @@ export function getInputTerm(
     // (2020/12/9): When a new timeline project is created, phases will initially
     // be []. To make sure we don't break copy that depends on an input_term,
     // we have the fallback to idea here in that case.
-    timeline:
-      !isNilOrError(phases) && phases.length > 0
-        ? getPhaseInputTerm(phases)
-        : 'idea',
+    timeline: !isNilOrError(phaseFromUrl)
```

Comment:
Minor: It might be best to avoid nested ternaries https://eslint.org/docs/latest/rules/no-nested-ternary